contractcourt/breacharbiter_test.go

```diff
@@ -956,18 +956,17 @@ restartCheck:
 
 func initBreachedState(t *testing.T) (*BreachArbiter,
 	*lnwallet.LightningChannel, *lnwallet.LightningChannel,
-	*lnwallet.LocalForceCloseSummary, chan *ContractBreachEvent,
-	func(), func()) {
+	*lnwallet.LocalForceCloseSummary, chan *ContractBreachEvent) {
 	// Create a pair of channels using a notifier that allows us to signal
 	// a spend of the funding transaction. Alice's channel will be the on
 	// observing a breach.
-	alice, bob, cleanUpChans, err := createInitChannels(t, 1)
+	alice, bob, err := createInitChannels(t, 1)
```

Comment:
That's because `sweep/store_test.go` uses `channeldb.MakeTestDB` and `contractcourt` uses `lnwallet.CreateTestChannels`. I had to combine these changes otherwise the **check commits** CI will fail: https://github.com/lightningnetwork/lnd/runs/8048766250